build/Microsoft.DotNet.Cli.Version.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+    <PropertyGroup>
+      <VersionMajor>2</VersionMajor>
+      <VersionMinor>0</VersionMinor>
+      <VersionPatch>0</VersionPatch>
+      <ReleaseSuffix>alpha</ReleaseSuffix>
+      <Channel>master</Channel>
+      <BranchName>master</BranchName>
+
+      <CliVersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionPatch)-$(ReleaseSuffix)</CliVersionPrefix>
+      <SimpleVersion>$(VersionMajor).$(VersionMinor).$(VersionPatch).$(CommitCount)</SimpleVersion>
+      <VersionSuffix>$(ReleaseSuffix)-$(CommitCount)</VersionSuffix>
+      <NugetVersion>$(VersionMajor).$(VersionMinor).$(VersionPatch)-$(VersionSuffix)</NugetVersion>
+      
+      <SdkVersion>$(NugetVersion)</SdkVersion>
```

Comment:
This one isn't needed either. They are all just dups of NuGetVersino. I ran into this on my DropSuffix change, but didn't feel like removing it as part of that change.